frontends/web/src/routes/account/account.tsx

```diff
@@ -186,7 +189,7 @@ export function Account({
     }
   }
 
-  const showBuyButton = moonpayBuySupported
```

Comment:
This is a small bug I found: the buy button logic was tied only to Moonpay support. Now it checks the availability among all the exchanges.